TS 6.0: Property 'readyState' in type 'WebSocketOverride' is not assignable to the same property in base type 'WebSocket'

I've tried using `typescript@6.0.0-beta` and I get this error:


```
error TS2416: Property 'readyState' in type 'WebSocketOverride' is not assignable to the same property in base type 'WebSocket'.
  Type 'number' is not assignable to type '0 | 1 | 3 | 2'.

85     readyState: number;
       ~~~~~~~~~~
```

Related code: https://github.com/mswjs/interceptors/blob/35d5188e3bda34a6997e4347b06d05f234011c17/src/interceptors/WebSocket/WebSocketOverride.ts#L33

--- a/src/interceptors/WebSocket/WebSocketOverride.ts
+++ b/src/interceptors/WebSocket/WebSocketOverride.ts
@@ -30,7 +30,7 @@ export class WebSocketOverride extends EventTarget implements WebSocket {
   public protocol: string
   public extensions: string
   public binaryType: BinaryType
-  public readyState: number
+  public readyState: WebSocket['readyState']
   public bufferedAmount: number
 
   private _onopen: WebSocketEventListener | null = null

